cirrus/server/cirrus/experiment_recipes.py

```diff
@@ -0,0 +1,13 @@
+from typing import Any, Dict, List
```

Comment:
Oh this is why I was saying to set the typing to the strictest, so we wouldn't ever have `Any` be allowed.  Force us to used `TypedDict` or Pydantic or whatever to explicitly lay out the structures of data.

Reply:
so right now pyright is in strict mode, for more context- Pyright differentiates between explicit and implicit forms of Any. The implicit form is referred to as [Unknown](https://github.com/microsoft/pyright/blob/main/docs/type-inference.md#unknown-type). For example, if a parameter is annotated as list[Any], that is a use of an explicit Any, but if a parameter is annotated as list, that is an implicit Any, so pyright refers to this type as list[Unknown]. Pyright implements several checks that are enabled in “strict” type-checking modes that report the use of an Unknown type. Such uses can mask type errors.

Mypy does not track the difference between explicit and implicit Any types, but it supports various checks that report the use of values whose type is Any: --warn-return-any and --disallow-any-*. For details, refer to [this documentation](https://mypy.readthedocs.io/en/stable/command_line.html#disallow-dynamic-typing).

Pyright’s approach gives developers more control. It provides a way to be explicit about Any where that is the intent. When an Any is implicitly produced due to an missing type argument or some other condition that produces an Any within the type checker logic, the developer is alerted to that condition.
So I will file a separate ticket and look into more how to do that, but right now `Any` is just a place holder which will be replaced by data classes

Reply:
Thanks for the excellent explanation @yashikakhurana 🙏

Reply:
also follow up ticket- https://github.com/mozilla/experimenter/issues/8762